repository-data/webfiles/src/main/resources/site/freemarker/common/scripts/document-search.js.ftl

```diff
@@ -0,0 +1,113 @@
+<#ftl output_format="HTML">
+
+<script>
+    $(document).ready(function() {
```

Comment:
Are there any acceptance tests for this new JavaScript?

Reply:
As far as I know we don't have any unit tests for the existing JavaScript code in the project, and I haven't created any new ones.

Reply:
I did not ask about unit tests. Good to know QA are adding acceptance tests.

Reply:
@GeoffHayward Acceptance tests cover the search, filter and highlight functionality as per the comment on the Jira ticket -> https://jira.digital.nhs.uk/browse/DW-1163?focusedCommentId=580209&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-580209

Reply:
I should have been more clear, are they covered by automated Cucumber acceptance tests?